### PR TITLE
Feature/interface

### LINKED ISSUE #14: Drive Team Input
### 🚀 Summary:

Provide potential drive team commands and hide implementation details of the controllers.

### 📋 Tasks:

- [x] Create drive team file for pilot and operator classes/objects
- [x] Create functions to read drive team commands

### PR DESCRIPTION
**Pull request by magnite5**
### 🛠 Proposed Changes:

Drive team controls, dashboard support

### 📝 Details:

- Supports 2 xbox controllers
- Simple debug outputs to dashboard
- Addressed comments from previous pull request

### 🔗 Related Issue(s):

- Resolves #14

### CODE REVIEW

File: .pre-commit-config.yaml
Comment (NathanGrenier) [inline .pre-commit-config.yaml:1]: Are we going ahead and deleting the config?

Comment (Portnord) [reply .pre-commit-config.yaml:1]: Yes, delete,. We don't want pre commit checks. We should be able to commit any work in progress to our feature or bug branches.

Comment (NathanGrenier) [reply .pre-commit-config.yaml:1]: It is just a config, you only get the pre-commit hook if you use pre-commit the install command.

Do you still want to get rid of it?

File: robot.py [hunk 1]
```diff
@@ -1,7 +1,45 @@
...
+    self.hookretract = False
+    self.hookleft = False
+    self.hookright = False
+
+    self.aim = 0.0
+    self.fire = 0.0
+    self.unjam = False
+    self.pickup = 0.0
+    self.eject = False
+    wpilib._wpilib.TimedRobot.__init__(self)
```
[the remark this answers is not in the document]
Comment (Portnord) [reply robot.py:25]: Required by the framework. It will throw an error and end the program without it.

[the remark this answers is not in the document]
Comment (NathanGrenier) [reply robot.py:25]: True, because you're overriding the constructor.

**Review by NathanGrenier — APPROVED** [review]

**Review by Portnord — APPROVED** [review]

**Pull request merged**